CHANGES.md

```diff
@@ -12,8 +12,9 @@ Format of the entries must be.
 <new-line>
 * Entry two with no-newlines. (DCOS_OSS_JIRA_2)
 ```
+### Notable Changes
 
-* Updated to [DC/OS UI 1.10.9-rc.1](https://github.com/dcos/dcos-ui/releases/tag/v1.10.9-rc.1)
+* Updated to [DC/OS UI 1.10.9-rc2](https://github.com/dcos/dcos-ui/releases/tag/1.10+v1.10.9-rc2)
```

Comment:
Shall we stick to having `v` in front of the tag name? `v1.10+v1.10.9-rc2`

Reply:
That is not the pattern we followed on the examples I saw on github:
https://github.com/dcos/dcos-ui/releases/tag/1.11%2Bv1.19.0

If we wanna do that, then we should document it on the release pattern. I prefer not having because for master it breaks the pattern.

For this bump I would say no.